spec/factories/site_statuses.rb

```diff
@@ -22,6 +22,7 @@
     transient do
       any_vancancy { false }
       provider { build :provider }
+      age { nil }
```

Comment:
whats the `age` for?

Reply:
It's a remnant of a fix I was attempting, I ended up just going with TimeCop in the end.